guides/common/modules/ref_supported-clients-in-registration.adoc

```diff
@@ -0,0 +1,79 @@
+[id="supported-clients-in-registration_{context}"]
+= Supported Clients in Registration
+
+{Project} supports the following operating systems and architectures for registration.
+
+ifndef::orcharhino,satellite[]
+[NOTE]
+====
+The following combinations have been tested.
+https://github.com/theforeman/foreman/blob/develop/app/views/unattended/provisioning_templates/registration/global_registration.erb[*Global registration*] is a provisioning template for a shell script, which can be extended for additional systems.
+
+If you decide to extend the template, please submit your changes to https://github.com/theforeman/foreman/#how-to-contribute[our repository].
+Thanks for your contribution!
+====
+endif::[]
+
+Supported Host Operating Systems::
+The hosts can use the following OS versions:
+
+ifndef::satellite[]
+* AlmaLinux
+* Amazon Linux
+* CentOS
+endif::[]
+ifndef::satellite[]
+* Debian
+ifndef::orcharhino[]
+* Fedora
+* FreeBSD
+* OpenSUSE
+endif::[]
+* Oracle Linux
+endif::[]
+* {EL} 9, 8, 7
+ifdef::satellite[]
+* {EL} 6 with the https://www.redhat.com/en/resources/els-datasheet[ELS Add-On]
+endif::[]
+ifndef::satellite[]
+* {SLES}
+ifndef::orcharhino[]
+* Solaris
+endif::[]
+* Ubuntu
+endif::[]
+
+Supported Host Architectures::
+The hosts can use the following architectures:
+
+[cols="2,3"]
+|===
+|Operating System |Architectures
+
+ifdef::satellite[]
+|{EL}
+|i386, x86_64, s390x, ppc_64
+endif::[]
+
+ifdef::orcharhino[]
+|Debian 10
+|aarch64, ppc64le, x86_64, amd64
```

Comment:
```suggestion
|aarch64, ppc64le, amd64
```

The architectures are named differently between EL and Debian. See https://wiki.debian.org/SupportedArchitectures

Reply:
I see! Thank you